se.sjoerd.GIScan.appdata.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>se.sjoerd.GIScan</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <name>GIScan</name>
+  <summary>Simple tool to read and analyze GISAXS data in cbf format</summary>
+  <developer_name>Sjoerd Broekhuijsen</developer_name>
+
+  <description>
+    <p>
+      GIScan is a simple tool to open and analyze data stored in cbf (Crystallographic Binary Format) files. While
+      GIScan is primarily developed with the analyis of GISAXS data in mind, as obtained by the P03 beamline of PETRA
+      III at the DESY Synchrotron in Hamburg, Germany, it should be compatible with any arbitrary cbf data.
+      </p>
+    <p>
+      GIScan handles raw data directly and doesn't need any a-priori data reduction. It supports features such as
+      background subtraction, line scans along the Qy and Qz axis, detection of the full-width at half-maximum (FWHM)
+      of diffraction peaks and coordinate conversion from pixels to reciprocal or angular space. </p>
+    <p>
+      GIScan is mostly developed with the analysis of multilayers in mind but can be used for any arbitrary data stored
+      in cbf format. If a beamline or datatype (be it GISAXS or GISANS) is not supported and you would like to use this
+      tool, please submit an issue on the Github page so this can be arranged.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">se.sjoerd.GIScan.desktop</launchable>
+
+  <url type="bugtracker">https://github.com/SjoerdB93/GIScan/issues</url>
+  <url type="help">https://github.com/SjoerdB93/GIScan/</url>
+  <url type="homepage">https://www.sjoerd.se</url>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>GIScan main view</caption>
+      <image width="3710" height="2100">http://www.sjoerd.se/giscan_main.png</image>
```

Comment:
There's no need to define width and height.

Reply:
Thank you, I removed the width and height attributes. Worked OK on my local test build without them.